core/jvm/src/main/scala/scalaz/zio/internal/impls/RingBuffer.scala

```diff
@@ -9,7 +9,7 @@ object RingBuffer {
   /**
    * @note mimimum supported capacity is 2
    */
-  def apply[A](requestedCapacity: Int): RingBuffer[A] = {
+  final def apply[A](requestedCapacity: Int): RingBuffer[A] = {
```

Comment:
aren't object methods final already? afaik you can't extend from an object
or is there other purpose to make them `final`?